src/cryptenroll/cryptenroll-fido2.c

```diff
@@ -11,7 +11,8 @@ int enroll_fido2(
                 struct crypt_device *cd,
                 const void *volume_key,
                 size_t volume_key_size,
-                const char *device) {
+                const char *device,
+                bool lock_with_pin) {
```

Comment:
if you add this, i'd prefer if we'd hook up all three of "clientPin", "up" and "uv" with user configuration and store the setting in JSON, a the same time (and then make this a flags param)

"up" and "uv" should probably encoded as tristate in JSON, i.e. true/false/null, apping to libfido2's FIDO_OPT_TRUE/FIDO_OPT_FALSE/FIDO_OPT_OMIT